core/src/main/scala/io/chrisdavenport/rediculous/RedisConnection.scala

```diff
@@ -194,18 +194,32 @@ object RedisConnection{
         },
         { case (_, shutdown) => shutdown}
       ).build
+
+      // Cluster Topology Acquisition and Management
       sockets <- Resource.liftF(keypool.take((host, port)).map(_.value._1).map(DirectConnection(_)).use(ClusterCommands.clusterslots[Redis[F, *]].run(_)))
-      refTopology <- Resource.liftF(Ref[F].of(sockets))
+      now <- Resource.liftF(Sync[F].delay(java.time.Instant.now))
```

Comment:
❓ Why not `Clock[F].realtime`? Using `Instant` will prevent anyone from using `TestContext` safely if their test code calls this

Reply:
I was intending to eventually probably do this via cats-effect-time. Which would do as you say.